From 3ce7bf419e366d88a141d1210ae96182986ea8e9 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Wed, 9 Jun 2021 15:47:14 +0200
Subject: [PATCH] meson: fix gcc vector 64-bit check

the previous behavior only ever enabled gcc vectors no x86_64
patch origin: https://github.com/ebassi/graphene/pull/233
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 5dbfb63..fd37036 100644
--- a/meson.build
+++ b/meson.build
@@ -328,7 +328,7 @@ if get_option('gcc_vector')
 #   error "GCC vector intrinsics are disabled on GCC prior to 4.9"
 # elif defined(__arm__)
 #   error "GCC vector intrinsics are disabled on ARM"
-# elif !defined(__x86_64__)
+# elif (__SIZEOF_POINTER__ < 8)
 #   error "GCC vector intrinsics are disabled on 32bit"
 # endif
 #else
